udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -385,17 +405,29 @@ const formatSpatialZones = (data: Array<SpatialZone>) => {
     }
   });
 return suggestions;
-}
+};
 
 const notUnknown = not(t("The value must be different than unknown."), sameAs("unknown"));
 const tagsRequired = requiredWithCustomMessage(t("Adding tags helps improve the SEO of your data."));
 const temporalCoverageRequired = requiredWithCustomMessage(t("You did not provide the temporal coverage."));
 const spatialGranularityRequired = requiredWithCustomMessage(t("You have not specified the spatial granularity."));
+const isSelectedProducer = ref<boolean>(false);
+
+function checkOrganization() {
+  return isSelectedProducer.value;
```

Comment:
This variable is doing the same thing as the vuelidate `$dirty` state
```suggestion
  return vWarning$.value.userOrganization.$dirty;
```

Reply:
Trying this change triggers an error

Reply:
This should be updated to the new field name. Also, depending on how Vue deal with function in script setup, you can try to move the call to `useFunctionalState` above this one